### ISSUE TITLE
Redis recipe: add the option to configure a Redis slave instance

### ISSUE THREAD

**Issue by radamanthus** [engineyard/ey-cookbooks-stable-v5#134]
Please modify the Redis recipe to support booting a Redis slave instance. Add the [SLAVEOF](https://redis.io/commands/slaveof) directive in the configuration file for the Redis slave instance.

**Comment by radamanthus** [comment]
TEST CASES

1) Normal case

Boot a cluster with 2 utility instances, 'redis' and 'redis_slave'

Modify the custom-redis recipe. Uncomment these lines in `attributes/default.rb`

```
#redis['slave_name'] = 'redis_slave'
#redis_instances << redis['slave_name']
```

Upload the cookbooks and and run chef

Verify that redis is running on both redis and redis_slave instances

Use the Redis CLI to store some values on the redis instance

Verify that the values are replicated to the redis_slave instance

Terminate the redis instance
Document the steps needed to convert redis_slave into a redis master instance

2) Edge cases

2.1. Chef should not fail even if redis slave instance is gone
Rename the 'redis_slave' instance to 'unrecognized_redis_slave'
Upload and run chef. Verify that there are no chef errors

2.2. Chef should not fail on a new environment even if there is no redis slave instance

**Comment by radamanthus** [comment]
This was addressed in PR #136
